R/navbar.R

```diff
@@ -176,3 +193,97 @@ pkg_navbar_vignettes <- function(
 
   tibble::tibble(name = name, title = title, file_out)
 }
+
+bs4_navbar_links_html <- function(links) {
```

Comment:
this and the functions below are part of what comes straight from #1111

Reply:
Can you restyle please?  And is there some reason you need both `bs4_navbar_links_html()` and `bs4_navbar_links_tags()`?

Reply:
`bs4_navbar_links_tags()` calls itself.